contracts/lib/SafeMathInt.sol

```diff
@@ -102,4 +102,13 @@ library SafeMathInt {
         require(a != MIN_INT256);
         return a < 0 ? -a : a;
     }
+
+    // This is a stub to be replaced with function that can compute 2^fraction
+    function twoPow(int256 exp, int256 one)
```

Comment:
Maybe just passing decimals makes more sense in the context of a library function 
```
function twoPow(int256 exp, int256 decimals) {
	return mul(one, 1<<div(exp,10**decimals));
}
```

Reply:
I don't think it matters much 'one' seem makes function code cleaner, check the full implementation now, let me now if you disagree